spec/rubocop/cop/graphql/ordered_fields_spec.rb

```diff
@@ -1,6 +1,28 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::GraphQL::OrderedFields, :config do
+  let(:config) do
+    RuboCop::Config.new(
+      "GraphQL/OrderedFields" => {
+        "Groups" => true
```

Comment:
we need a test with Groups = false as well 🙂

Reply:
Sounds good, I added a few but will add more!

Reply:
Hey @DmitryTsepelev I added some tests [here](https://github.com/DmitryTsepelev/rubocop-graphql/blob/365baa0b195d92e9d1c883655ff01e9970a897f7/spec/rubocop/cop/graphql/ordered_fields_spec.rb#L136) originally. Were these not what you expected?

Reply:
yep yep, you're totally right